### PR TITLE
Adds SUPPORT.md

### PR DESCRIPTION
**Pull request by mrjoro**
Adds a SUPPORT.md file to tell people how to get help using AMP.  This fixes issue #10567.

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+There are many ways to get help for questions and issues related to AMP:
```
Comment (bpaduch) [inline SUPPORT.md]: Recommend including a title

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- If you are looking for help **getting started using AMP on your site** or are **having issues using AMP**:
+  - the [ampproject.org FAQ](https://www.ampproject.org/support/faqs/) provides answers to common questions about AMP
+  - [Stack Overflow](http://stackoverflow.com/questions/tagged/amp-html) is our recommended way to find answers to questions about AMP; since members of the AMP Project community regularly monitor Stack Overflow you will probably receive the fastest response to your questions here
```
Comment (bpaduch) [inline SUPPORT.md]: replace "here" with "there"

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  - the [amphtml-discuss Google Group](https://groups.google.com/forum/#!forum/amphtml-discuss) is another resource for finding answers to AMP related questions
+  - [ampbyexample.com](https://ampbyexample.com) provides hands-on tutorials for using AMP components
+  - [ampstart.com](https://ampstart.com) provides templates you can use to create AMP sites from scratch
```
Comment (bpaduch) [inline SUPPORT.md]: add "pre-styled" before templates

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+There are many ways to get help for questions and issues related to AMP:
+
+- If you are looking for help **getting started using AMP on your site** or are **having issues using AMP**:
```
Comment (bpaduch) [inline SUPPORT.md]: Suggest consistent capitalization for first word of each bullet

File: SUPPORT.md (hunk 1 above)
Comment (mrjoro) [inline SUPPORT.md]: Done.

File: SUPPORT.md (hunk 4 above)
Comment (mrjoro) [inline SUPPORT.md]: Done.

File: SUPPORT.md (hunk 2 above)
Comment (mrjoro) [inline SUPPORT.md]: Done.

File: SUPPORT.md (hunk 3 above)
Comment (mrjoro) [inline SUPPORT.md]: Done.

**Pull request merged**